ops/ansible/roles/bfd-server/templates/newrelic.yml.j2

```diff
@@ -0,0 +1,18 @@
+# This file configures the New Relic Agent.  New Relic monitors
+# Java applications with deep visibility and low overhead.  For more details and additional
+# configuration options visit https://docs.newrelic.com/docs/agents/java-agent/configuration/java-agent-configuration-config-file.
+
+transaction_events:
+  custom_request_headers:
+    - header_name: "BlueButton-OriginalQueryId"
+    - header_name: "BlueButton-OriginalQueryId"
+    - header_name: "BlueButton-OriginalQueryCounter"
+    - header_name: "BlueButton-OriginalQueryTimestamp"
+    - header_name: "BlueButton-DeveloperId"
+    - header_name: "BlueButton-Developer"
+    - header_name: "BlueButton-ApplicationId"
+    - header_name: "BlueButton-Application"
+    - header_name: "BlueButton-UserId"
+    - header_name: "BlueButton-User"
+    - header_name: "BlueButton-BeneficiaryId"
+    - header_name: "x-forwarded-for"
```

Comment:
Does this get merged into another config file or something? From a quick googling this appears to be a partial config. (Sorry, not yet familiar with how our new relic agent is configured)

Reply:
It's definitely a partial file. Which I'm  hoping is fine, because the [full template is quite long](https://docs.newrelic.com/docs/agents/java-agent/configuration/java-agent-config-file-template/). A lot of those fields are optional and/or have defaults that we don't override with the current environment variables. And those environment variables override values that we _would_ put in those files.

My plan was to deploy this to test before going anywhere with it to help answer that question. If we need the whole file, then I'll fix up the whole file and stop configuring it in a split-fashion between this and env vars.